lib/iris/tests/test_coordsystem.py

```diff
@@ -33,17 +33,12 @@
 from iris.coord_systems import *
```

Comment:
Wow. These are some really old tests (and really old-school style). 😉

Reply:
Whoa 😨